packages/@aws-cdk/region-info/build-tools/fact-tables.ts

```diff
@@ -156,5 +159,30 @@ export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
 
   'me-south-1': '772975370895',
   'ap-east-1': '856666278305',
+};
 
+/**
+ * Latest Envoy Image released by the App Mesh Team
+ *
+ * @see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+ */
+export const APPMESH_LATEST_ENVOY_IMAGES: { [region: string]: string} = {
```

Comment:
`LATEST` here is (or will eventually become) a lie...

Reply:
Yeah, I agree. Any reason for not calling it just `APPMESH_ENVOY_IMAGES`?

Also, what are the consequences of updating these in the future? Does it have any chance of causing downtime for customers that update?

Reply:
So we have thought about moving the envoy images to SSM in the future. https://github.com/aws/aws-app-mesh-roadmap/issues/257

Would this be a better path forward? I would be ok closing this PR for now. 

We release envoy images that have been vetted to work with our service. Updating them should not cause downtime unless there is a bug with the image.

Reply:
I think that's fine if you want to close this one @dfezzie , you can always re-open it later if you need to 🙂